.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk-version: [adopt@1.11]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Scala ${{ matrix.jdk-version }}
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: ${{ matrix.jdk-version }}
```

Comment:
It appears that Jabba is being used under the covers by this Action. If we want to test against additional JDK versions, we can add them to the `jdk-version` matrix list above.